Description: Fix wheel.install doesn't exist anymore
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Bug-Debian: https://bugs.debian.org/917678
Last-Update: 2018-12-30

--- python-pbr-4.2.0.orig/pbr/tests/test_packaging.py
+++ python-pbr-4.2.0/pbr/tests/test_packaging.py
@@ -56,7 +56,10 @@ import testscenarios
 import testtools
 from testtools import matchers
 import virtualenv
-import wheel.install
+try:
+    import wheel.install as wheel_file_class
+except ImportError:
+    import wheel.wheelfile as wheel_file_class
 
 from pbr import git
 from pbr import packaging
@@ -372,7 +375,7 @@ class TestPackagingWheels(base.BaseTestC
         relative_wheel_filename = os.listdir(dist_dir)[0]
         absolute_wheel_filename = os.path.join(
             dist_dir, relative_wheel_filename)
-        wheel_file = wheel.install.WheelFile(absolute_wheel_filename)
+        wheel_file = wheel_file_class.WheelFile(absolute_wheel_filename)
         wheel_name = wheel_file.parsed_filename.group('namever')
         # Create a directory path to unpack the wheel to
         self.extracted_wheel_dir = os.path.join(dist_dir, wheel_name)
